bin/smack.py

```diff
@@ -15,13 +15,71 @@
 import sys
 import tempfile
 from threading import Timer
+from toSVCOMPformat import *
+from token_replace import *
 
 VERSION = '1.5.1'
 temporary_files = []
 
+def rewriteIExtensionToC(args):
```

Comment:
Ok, so I talked to Shaobo, and I found out more about this...  

So we are including smack.h to benchmarks not by injecting `#include <smack.h>` into the benchmark, but rather by using the clang switch `-Ismack.h` (or currently, `-Ismack-svcomp.h`).  This is preprocessor functionality.

When clang sees the file is a `.i` file, it disables the preprocessor, which means that even files included with the `-I` switch don't get included.

A cursory search turned up ways to *disable* the preprocessor, but I haven't found a way to force it on.

At this point, I think it might be worth spending a little more time to look for a way to force the preprocessor on, but I haven't found a solution yet.  I just wanted to add this comment as a status update with some additional info.